packages/mixpanel/src/utils.ts

```diff
@@ -47,13 +47,13 @@ export function getUTMParamsFromSearch(searchString: string): UTMParams | undefi
 
 const platform = env('SCOUTGAME_PLATFORM') || process.env.REACT_APP_SCOUTGAME_PLATFORM;
 
-function isPlatform(_platform: string = ''): _platform is ReferralPlatform {
-  const availablePlatforms = Object.values(ReferralPlatform);
+function isPlatform(_platform: string = ''): _platform is ReferralPlatform | 'onchainwebapp' {
```

Comment:
please extend ReferralPlatform with 'onchain-webapp' instead of adding this separately